gems/laa_multi_step_forms/app/views/layouts/_phase_banner.html.erb

```diff
@@ -1,14 +1,2 @@
-<div class="govuk-phase-banner" role="complementary">
-  <p class="govuk-phase-banner__content">
-    <strong class="govuk-tag govuk-phase-banner__content__tag">
-      <% if ENV.fetch('ENV', 'local') == 'production' %>
-        <%= t('.phase') %>
-      <% else %>
-        <%= ENV.fetch('ENV', 'local') %>
-      <% end %>
-    </strong>
-    <span class="govuk-phase-banner__text">
-      <%= t('.phase_description', feedback: main_app.about_feedback_index_path).html_safe %>
-    </span>
-  </p>
-</div>
+<%= govuk_phase_banner(tag: { text: phase_name },
+                       text: t('.phase_description', feedback: main_app.about_feedback_index_path).html_safe) %>
```

Comment:
Nit: `html_safe`ing the output of `t` can occasionally get you into trouble, as it raises the possibility of XSS attacks if a user-entered string is ever passed in as an interpolation variable.

It's a good habit to get into to use `t('.blah_html', x: y)` instead of `t('.blah', x: y).html_safe`, as it will mean only the static template, not the interpolated variables, will be marked as safe.

I do appreciate that this particular line has no possibility of XSS and that you're only reusing the code that was already there, though!

Reply:
done, good spot